components/filter/filter.js

```diff
@@ -716,6 +744,37 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		e.stopPropagation();
 	}
 
+	_updateActiveFilters() {
+		const activeFilters = [];
+
+		this._dimensions.forEach(dimension => {
+			switch (dimension.type) {
+				case 'd2l-filter-dimension-set': {
+					dimension.values.forEach(value => {
+						if (value.selected) {
+							const keyObject = { dimension: dimension.key, value: value.key };
+							const text = dimension.valueOnlyActiveFilterText ? value.text : this.localize('components.filter.activeFilterText', { filterName: dimension.text, activeFilter: value.text });
```

Comment:
I'm not totally convinced that this needs to be localized, since it's just one thing before the other separated by a colon. My worry is just that it's one more place where translators could mess it up... and I can't think of any language-specific variances that there would be anyway.

Reply:
Yeah I agree, we can always add a langterm later if it gets more complicated but for now it's only asking for trouble